plugins/GSdx/GSHwHack.cpp

```diff
@@ -750,6 +750,7 @@ bool GSC_TalesOfLegendia(const GSFrameInfo& fi, int& skip)
 	return true;
 }
 
+//Shadows don't appear or cause darkness issues in different levels.
```

Comment:
This comment is better placed inside the CRC hack.

Reply:
"causes" or "may cause".

Reply:
The wording is fine.

Reply:
Well if we are gonna change the wording it should be.

"// Removes depth effects not rendered correctly on all renders."